client/rest-high-level/src/test/java/org/elasticsearch/client/ml/job/results/AnomalyRecordTests.java

```diff
@@ -38,6 +38,7 @@ public static AnomalyRecord createTestInstance(String jobId) {
         anomalyRecord.setActual(Collections.singletonList(randomDouble()));
         anomalyRecord.setTypical(Collections.singletonList(randomDouble()));
         anomalyRecord.setProbability(randomDouble());
+        anomalyRecord.setImpact(randomDouble());
```

Comment:
To confirm `null` works once the type is changed to `Double`, surround this line with `if (randomBoolean()) {`.

Same for the server-side file.

Reply:
Will do - thanks for the quick review @droberts195